swagger_server/itm/itm_session.py

```diff
@@ -447,7 +452,7 @@ def start_session(self, adm_name: str, session_type: str, adm_profile: str, kdma
                 self._end_session() # Exception here ends the session
                 return 'Exception communicating with TA1; is the TA1 server running?  Ending session.', 503
 
-        path = f"swagger_server/itm/data/{ITMSession.EVALUATION_TYPE}/" + ("test/" if self.session_type == 'test' else 'scenarios/')
+        path = f"{ITMSession.SCENARIO_DIRECTORY}" + ("test/" if self.session_type == 'test' else 'scenarios/')
```

Comment:
A few things:
- In the ticket, we talked about Test scenarios always being in `itm/data/{evaluation}/test`.  Those are used during `test` sessions.  So use `EVALUATION_TYPE` but not `SCENARIO_DIRECTORY` for `test` sessions.
- It seems we could add a `/` after `{ITMSession.SCENARIO_DIRECTORY}` so that their paths work whether or not they specify a trailing slash.
- Maybe we shouldn't require their paths have a `scenarios` subdirectory.  In other words, if they specify `foobar`, we should look for scenarios in `./foobar/` instead of `./foobar/scenarios/`.  The default would need to change to `swagger_server/itm/data/%(EVALUATION_TYPE)s/scenarios/`.